### ISSUE TITLE
Minimum width and height in pixels

### ISSUE THREAD

**Issue by monsieurtanuki** [deakjahn/crop_image#30]
In my app, I'm not supposed to crop into too small a picture, because the cropped image will then be rejected by the server I upload to.
In my particular case, we have a minimum width and a minimum height. It's even more complicated, the dimensions are ok if `width >= minimumWidth` OR `height >= minimumHeight`.

How could we implement those limits in crop_image?
Perhaps we should add a method to the controller, null by default, that checks if the dimensions are OK and recomputes the new dimensions if needed.
A method to be implemented by the app developer if needed in each specific case, for more flexibility.

cf. https://github.com/openfoodfacts/smooth-app/issues/4219

**Comment by deakjahn** [comment]
Yes, an additional field (maybe not a method, just a field) could be good.
